src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -33,10 +40,43 @@
   /** Create a new Businesses instance
   * @param businesses businesses from SmallCityService
   **/
+  
   public BusinessesService(List<Listing> businesses) {
     this.businesses = businesses;
   }
 
+  public PreparedQuery getBigBusinessFromDatabase(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("BigBusinesses");
+    PreparedQuery dateabaseResults = datastore.prepare(query);
+    return dateabaseResults;
```

Comment:
Should "dateabaseResults" be "databaseResults"? 

Also, note this query isn't executed at this point just yet. This makes "databaseResults" misleading. The execution / retrieval doesn't happen until asIterablae() is called in removeBigBusinessFromResults() below. To make this clear, perhaps rename this to be a bit more specific. How about something like preparedQueryForBigBusinesses? 

ref: https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/PreparedQuery.html#asiterable

Reply:
fixed both issues